app/views/admin/bikes/missing_manufacturer.html.haml

```diff
@@ -3,10 +3,13 @@
     %h1
       Multi manufacturer update
   .col-md-6
-    -if session[:missing_manufacturer_time_order]
-      = link_to "Normal view", missing_manufacturer_admin_bikes_path(reset_view: true), class: "btn btn-primary"
-    -else
-      = link_to "Time ordered view", missing_manufacturer_admin_bikes_path(time_ordered: true), class: "btn btn-primary"
+    %ul
+      -if session[:missing_manufacturer_time_order]
```

Comment:
I'd prefer if both of these showed all the time, and the one that was currently being viewed ("alphabetical ordering" or "creation ordering") had the active class.